src/Docfx.Common/Git/GitUtility.cs

```diff
@@ -38,9 +39,13 @@ record Repo(string path, string url, string branch);
         if (repo is null)
             return null;
 
+        var repoUrl = EnvironmentContext.IsSnapshotTestContext
```

Comment:
Can repo URL be configured from the `DOCFX_SOURCE_REPO_URL` environment similar to how `DOCFX_SOURCE_BRANCH_NAME` works (and how branch are unified for PR builds)?